dev/scripts/src/alluxio.org/build-distribution/cmd/generate-tarball.go

```diff
@@ -301,8 +357,20 @@ func generateTarball(hadoopDistribution string) error {
 	run("adding Alluxio server assembly jar", "mv", fmt.Sprintf("assembly/server/target/alluxio-assembly-server-%v-jar-with-dependencies.jar", version), filepath.Join(dstPath, "assembly", fmt.Sprintf("alluxio-server-%v.jar", version)))
 	run("adding Alluxio FUSE jar", "mv", fmt.Sprintf("integration/fuse/target/alluxio-integration-fuse-%v-jar-with-dependencies.jar", version), filepath.Join(dstPath, "integration", "fuse", fmt.Sprintf("alluxio-fuse-%v.jar", version)))
 	run("adding Alluxio checker jar", "mv", fmt.Sprintf("integration/checker/target/alluxio-checker-%v-jar-with-dependencies.jar", version), filepath.Join(dstPath, "integration", "checker", fmt.Sprintf("alluxio-checker-%v.jar", version)))
-	// Condense the webapp into a single .war file.
-	run("jarring up webapp", "jar", "-cf", filepath.Join(dstPath, webappWar), "-C", webappDir, ".")
+
+	masterWebappBuildDir := "alluxio-ui/master/build"
+	run ("creating alluxio-ui master webapp directory and copying files", "mkdir", "-p", filepath.Join(dstPath, masterWebappBuildDir))
+	err = CopyDir(masterWebappBuildDir, filepath.Join(dstPath, masterWebappBuildDir))
```

Comment:
could this be simplified to one line without need of a help function?
`run ("Moving master webapp resources", "mv", masterWebappBuildDir, filepath.Join(dstPath, masterWebappBuildDir)`

Reply:
Yes, I will do, thanks!

Reply:
oops, style should be `run(`, not `run (`

Reply:
I got ya